### ISSUE TITLE
Epic: WAVE Alert: Orphaned form label

### ISSUE THREAD

**Issue by geolunalg** [hackforla/tdm-calculator#2896]
### Dependency
When the sub issues are completed, then this issue will be resumed
- #3037 
- #3038 
- #3039

### Overview
We need to make issues that will create, apply and document a solution for Orphaned form label so that it can get fixed across the site.

### Details
Category: Alert
Name: Orphaned form label

known pages (there might be others):
1. Create Project Page 4
2. Security
3. Archived Accounts

This epic will do the following
- create issues to fix any occurrences of this problem

### Action Items
- [x] Go to the Solutions Index (resource 1.02) and add the link to the specific page for this problem 1.02.01
- [x] Copy the issue template (resource 1.04)
- [x] Use the template content to create a new issue(s) for the pages under Known pages above
- [x] Customize the draft
   - [x] Add the following labels
      - role: front-end
      - size: missing
      - p-feature: [REPLACE WITH FIRST PAGE NAME]
      - feature: Accessibility
      - level: missing
      - deck: add to staging
      - ready for accessibility lead
      - priority: MUST HAVE
   - Customize the draft issue resoures
      - [x] In this issue use the content in resource Product Documentation 1.03.01 to update the draft's issue resource 1.02.01, 1.02.02
      - [x] In this issue use the content in resource 1.01.01 (spreadsheet) to update the link to the spreadsheet in draft issue's resource 1.03
         - [x] open the spreadsheet link and find the row number, and add it to the draft issues's resource 1.03.01
- [x] In this issue, add issue to Sub Issue section below
- [ ] Once all the issues are closed, run WAVE and check all other pages for this alert; ignore all other alerts.  Check user interactions on the page to ensure that the alert isn't hidden by an interaction. Check one of the following below:
   - [ ] No additional instances of this alert found
   - [ ] New instances of these alerts found
      - [ ] Make additional issues

### Resources
- 1.01 [TDM WAVE Accessibility Audit Spreadsheet, tab: 2. Audit Log](https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=0#gid=0)
   - 1.01.01 [TDM WAVE Accessibility Audit Spreadsheet, tab: 2. Audit Log, filter: Alert: Orphaned form label](https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=0#gid=0&fvid=336057419)
- 1.02 [WCAG: Known Issues and Solutions](https://hackforla.github.io/accessibility/known-issues-and-solutions)
  - 1.02.01 https://hackforla.github.io/accessibility/known-issues-and-solutions/solutions-index/alert-orphaned-form-label
- 1.03 WIKI: Product Documentation: https://github.com/hackforla/tdm-calculator/wiki/Product-Documentation
   - 1.03.01 WIKI: Product Documentation, Create Projects page 4: https://github.com/hackforla/tdm-calculator/wiki/Create-Projects
   - 1.03.02 WIKI: Product Documentation, Security page: https://github.com/hackforla/tdm-calculator/wiki/Security
   - 1.03.03 WIKI: Product Documentation, Archived Accounts page: https://github.com/hackforla/tdm-calculator/wiki/Archive-Accounts

#### 1.04 Issue Template
Title
```
Dev: fix Alert: Orphaned form label on [REPLACE WITH PAGE NAME] page
```
Body
````
### Overview
We need to fix the Alert: Orphaned form label on [REPLACE WITH PAGE NAME] so that page can be WCAG compliant.

#### Details
Category: Alert
Name: Orphaned form label
(The Accessibility wiki for this Alert has a solution)

Page that will be worked on:
[REPLACE WITH NAME OF PAGE NAME]

### Action Items
- [ ] Review wiki page: Accessibility, How to run the WAVE browser extension (Resource 1.01)
   - [ ] Install the WAVE browser extension
- [ ] Log in as the user indicated (resource 1.02.01) 
- [ ] navigate to the page you are going to fix (resource 1.02.02)
- [ ] Open the TDM WAVE Accessibility Audit Spreadsheet, for this Alert (resource 1.03) and find the row that relates to this  issues page.
- [ ] Run the WAVE extension with no elements focused, or clicked
- [ ] Find the Alert referenced in the Spreadsheet (1.03)
- [ ] This alert has been resolved on other pages, so review `Project Team Solution` section (resource 1.04.01.01)
- [ ] Apply solution
- [ ] Submit PR
- [ ] Add PR to spreadsheet in PR column next to Alert (resource 1.03)
- [ ] On this issue
   - [ ] Add labels: 
      - `ready for accessibility lead`
   - [ ] Change status to: `Technical Review`
   - to the epic #2896
     - [ ] Add the label `ready for product`
      - [ ] Add this comment 
         ```
          - [#REPLACE WITH THIS ISSUE NUMBER] is closed.  Please review to see if there are any remaining instances of this problem
         ```

### Resources/Instructions
- 1.01[How to run the WAVE browser extension](https://github.com/hackforla/accessibility/blob/main/docs/getting-started/wave-browser-extension.md)
- 1.02 Application details
   - 1.02.01 User Profile Required: [REPLACE WITH USER TYPE: visitor (not logged in)/user/admin/security admin/]
   - 1.02.02 Page URL: http://tdm-dev.azurewebsites.net/[REPLACE WITH PAGE URL]
      - Requires sign in: [REPLACE WITH TRUE OR FALSE]
      - Can access page directly from URL: [REPLACE WITH TRUE OR FALSE]
- 1.03 [TDM WAVE Accessibility Audit Spreadsheet, tab: 2. Audit Log, filter: Alert: Orphaned form label](https://docs.google.com/spreadsheets/d/1qiAqR9LH-Gd4LO3aOs0yUy174o5LOmnuU6Bz671mKsY/edit?gid=0#gid=0&fvid=336057419)
   -  Spreadsheet Row(s): [REPLACE WITH ROW NUMBERS]
- 1.04 [Accessibility Known Issues Index](https://github.com/hackforla/accessibility/blob/main/docs/known-issues-and-solutions/solutions-index/index.md)
    - 1.04.01 [WAVE Known Accessibility Issue: Alert - Orphaned form label](https://hackforla.github.io/accessibility/known-issues-and-solutions/solutions-index/alert-orphaned-form-label)
        - 1.04.01.01 [WAVE Known Accessibility Issue: Alert - Orphaned form label, section: ProjectTeam Solution](https://hackforla.github.io/accessibility/known-issues-and-solutions/solutions-index/alert-orphaned-form-label/#project-team-solution)

Additional reading (optional)
- https://hackforla.github.io/accessibility/

#### Slides
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg)
   - [Staging Deck, slide ]
- Release deck
   - [2026-01-20 Release deck, slide ]
````

#### Slides
- [Staging Deck](https://docs.google.com/presentation/d/1crZ3IxqA4hAu3qzD7ns93Ieuqjwh6wyEtuX_46cP-fg)
   - [Staging Deck, slide 2897] - N/A
- Release deck
   - [2026-01-20 Release deck, slide 2897] - N/A
